plugins/aep-consent-configuration/plugin.json

```diff
@@ -0,0 +1,10 @@
+{
+  "category": "AEP Edge Consent",
+  "displayName": "Consent Default Status",
+  "description": "Validate default consent state status is y, n or p.",
```

Comment:
Title: Collect consent status (default settings)
Tooltip:  Checks if the Consent extension is installed in Data Collection Tag UI and the default collect consent level is set to yes (y), no (n) or pending (p). These settings apply for the events sent to the Edge Network.
Description(s):
	1. Yes: Default collect consent level is set to yes. Events are sent to the Edge Network.
	2. No: Default collect consent level is set to no. Events are dropped until the status is updated to yes or pending.
	3. Pending: Default collect consent level is set to pending. Events are queued until the status is updated to yes or no.
  4. Not set at all: Default collect consent level is not set. By default the collect consent settings used for Edge Network events is yes. If you intended to use the Consent extension to control these settings, please follow the steps in the link then reload this validator. https://developer.adobe.com/client-sdks/documentation/consent-for-edge-network/